packages/hydrogen-codegen/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@shopify/hydrogen-codegen",
+  "version": "0.0.0",
+  "private": true,
+  "description": "GraphQL Code Generator preset for gql magic.",
+  "type": "module",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.js",
+  "scripts": {
+    "build": "tsup --clean --config ./tsup.config.ts",
+    "dev": "tsup --watch --config ./tsup.config.ts",
+    "typecheck": "tsc --noEmit",
+    "test": "cross-env SHOPIFY_UNIT_TEST=1 vitest run",
+    "test:watch": "cross-env SHOPIFY_UNIT_TEST=1 vitest",
+    "postinstall": "node ./postinstall.mjs"
```

Comment:
FYI this wouldn't be needed in the final version if we can make a PR to `@graphql-tools` with the hooks we need.

Reply:
Post install is not needed anymore since now the dependency is patched when the CLI runs.